h2/stream.py

```diff
@@ -874,6 +901,9 @@ def advertise_alternative_service(self, field_value):
         Advertise an RFC 7838 alternative service. The semantics of this are
         better documented in the ``H2Connection`` class.
         """
+        self.config.logger.debug(
+            "Advertise alternative service of %s for %r", field_value, self
+        )
```

Comment:
Ahh foiled by python 3 again.  Noted. I need to pay more attention to this in general.